docs/writing-tests/test-addon.mdx

```diff
@@ -68,90 +68,7 @@ For some project setups, the `add` command may be unable to automate the addon a
-1. Adjust your Vitest configuration to include the plugin(s) and reference the setup file. You can use the [example configuration files](#example-configuration-files) as a guide.
-
-#### Framework plugins
-
-Some Storybook frameworks require additional setup to enable the framework's features to work with Vitest. Each of those frameworks exports a Vite plugin that you can use to configure your project correctly:
-
-<If renderer="react">
-  If you're using Next.js, first install the `@storybook/experimental-nextjs-vite` package:
-
-  {/* prettier-ignore-start */}
-
-  <CodeSnippets path="nextjs-vite-install.md" />
-
-  {/* prettier-ignore-end */}
-  
-  Then apply the plugin from `@storybook/experimental-nextjs-vite/vite-plugin`:
-
-  ```js title="vitest.config.ts"
-  import { defineConfig, mergeConfig } from 'vitest/config';
-  import { storybookTest } from '@storybook/experimental-addon-test/vitest-plugin';
-  import { storybookNextJsPlugin } from '@storybook/experimental-nextjs-vite/vite-plugin';
-
-  import viteConfig from './vite.config';
-
-  export default mergeConfig(
-    viteConfig,
-    defineConfig({
-      plugins: [
-        storybookTest(),
-        storybookNextJsPlugin(), // 👈 Apply the framework plugin here
-      ],
-      // ...
-    })
-  );
-  ```
-</If>
-
-<If renderer="vue">
-  Vue projects should apply the plugin from `@storybook/vue3-vite/vite-plugin`:
-
-  ```js title="vitest.config.ts"
-  import { defineConfig, mergeConfig } from 'vitest/config';
-  import { storybookTest } from '@storybook/experimental-addon-test/vitest-plugin';
-  import { storybookVuePlugin } from '@storybook/vue3-vite/vite-plugin';
-
-  import viteConfig from './vite.config'
-
-  export default mergeConfig(
-    viteConfig,
-    defineConfig({
-      plugins: [
-        storybookTest(),
-        storybookVuePlugin(), // 👈 Apply the framework plugin here
-      ],
-      // ...
-    })
-  );
-  ```
-</If>
-
-<If renderer="svelte">
-  If you're using SvelteKit, apply the plugin from `@storybook/sveltekit/vite-plugin`:
-
-  ```js title="vitest.config.ts"
-  import { defineConfig, mergeConfig } from 'vitest/config';
-  import { storybookTest } from '@storybook/experimental-addon-test/vitest-plugin';
-  import { storybookSveltekitPlugin } from '@storybook/sveltekit/vite-plugin';
-
-  import viteConfig from './vite.config';
-
-  export default mergeConfig(
-    viteConfig,
-    defineConfig({
-      plugins: [
-        storybookTest(),
-        storybookSveltekitPlugin(), // 👈 Apply the framework plugin here
-      ],
-      // ...
-    })
-  );
-  ```
-</If>
-
-The above example uses the framework's plugin in a Vitest configuration file. You can also use it in a Vitest workspace file, if that is how your project is configured.
+1. Adjust your Vitest configuration to include the plugin and reference the setup file. You can use the [example configuration files](#example-configuration-files) as a guide.
```

Comment:
I believe the `How do I apply custom Vite configuration?` FAQ will also need to be either rewritten to emphasize that we automatically pull everything from `viteFinal` or removed outright.

Reply:
Let me know if that's correct and you'd prefer I do it. Happy to help!

Reply:
sounds about right to me. Honestly, I'd love if you could do that, either in this PR or a follow-up, up to you.